snap/snapcraft.yaml

```diff
@@ -49,6 +49,7 @@ plugs:
   home:
     read: all
   removable-media:
+  raw-usb:
```

Comment:
not sure if this works or not, since the plugs declared "globally" under the global `plugs` get applied to:

* all apps/hooks which do not declare _any_ plugs 
* any apps/hooks which specifically declare this plug

So as it is, I think this would only apply to any hooks, since the `dockerd` and `docker` apps do not declare this plug and declare a different set of plugs. I have a question for the reporter though, so hold off on merging this first